src/calendar-backend/Calendar.Api/Controllers/LectureController.cs

```diff
@@ -0,0 +1,91 @@
+﻿using AutoMapper;
+using Calendar.Api.DTOs;
+using Calendar.Api.DTOs.Create;
+using Calendar.Api.DTOs.Update;
+using Calendar.Api.Services.Interfaces;
+using Calendar.Mongo.Db.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Calendar.Api.Controllers;
+
+[ApiController]
+[Route("v1/api/[controller]")]
+public class LectureController : ControllerBase
+{
+    private readonly ILectureService service;
+    private readonly IMapper mapper;
+    private readonly ILogger<LectureController> logger;
+
+    public LectureController(ILectureService service, IMapper mapper, ILogger<LectureController> logger)
+    {
+        this.service = service;
+        this.mapper = mapper;
+        this.logger = logger;
+    }
+
+    [HttpPost]
+    // [Authorize(Roles = "editor")]
+    public async Task<ActionResult<LectureDTO>> AddLecture([FromBody] CreateLectureDTO lecture)
+    {
+        if (lecture == null)
+        {
+            return BadRequest();
+        }
+
+        var result = await service.AddLectureAsync(mapper.Map<Lecture>(lecture));
+
+        return Ok(mapper.Map<LectureDTO>(result));
+    }
+
+    [HttpGet]
+    // [Authorize(Roles = "viewer,editor")]
+    public async Task<ActionResult<IEnumerable<LectureDTO>>> GetLecture(string lectureId)
```

Comment:
Was ist der Unterschied zum Endpunkt in Zeile 57?

Wir brauchen noch einen Endpunkt in den wir alle Lectures bekommen, (ohne ID). Nur sichtbar für Editoren.

Reply:
![image](https://github.com/mhummel-artiso/lecture-calendar/assets/97592367/01773c0b-b0cd-4480-9447-26360ddd361b)

Reply:
der hatte noch ein falschen parameter, der soll alle lectures zurück geben